tfjs-core/src/ops/conv2d_test.ts

```diff
@@ -228,6 +228,40 @@ describeWithFlags('conv2d', ALL_ENVS, () => {
     expectArraysClose(resultData, [20, 26, 13, 12]);
   });
 
+  it('x=[4,2,2] f=[2,2,4,4] s=1 d=1 p=same NCHW', async () => {
+    // Skip webgl backend due to bug
+    // https://github.com/tensorflow/tfjs/issues/6308.
+    if (tf.getBackend() === 'webgl') {
```

Comment:
@lina128 @pyu10055 I have skipped webgl backend. However, the bots still fail and report below error (https://console.cloud.google.com/cloud-build/builds/46a36ee9-8244-46ef-82de-0dd1b8a979b8;step=2?project=learnjs-174218):
```
Step #24 - "test-tfjs-node": 1) conv2d test-tensorflow {} x=[4,2,2] f=[2,2,4,4] s=1 d=1 p=same NCHW
Step #24 - "test-tfjs-node":   Message:
Step #24 - "test-tfjs-node":     Error: Invalid TF_Status: 12
Step #24 - "test-tfjs-node":     Message: The Conv2D op currently only supports the NHWC tensor format on the CPU. The op was given the format: NCHW
Step #24 - "test-tfjs-node":   Stack:
Step #24 - "test-tfjs-node":     Error: Invalid TF_Status: 12
Step #24 - "test-tfjs-node":         at <Jasmine>
Step #24 - "test-tfjs-node":         at NodeJSKernelBackend.executeSingleOutput (/workspace/tfjs-node/src/nodejs_kernel_backend.ts:189:41)
Step #24 - "test-tfjs-node":         at conv2dImpl (/workspace/tfjs-node/src/kernels/Conv2D.ts:78:18)
Step #24 - "test-tfjs-node":         at Object.kernelFunc (/workspace/tfjs-node/src/kernels/Conv2D.ts:37:12)
```
It seems that tfjs-node doesn't support NCHW format. But it's weird that others NCHW tests in this file don't report such kind of errors. Maybe it's because other cases's out channel is 1 which is treated as NHWC not NCHW?